lookout/style/typos/templates/comment.md.jinja2

```diff
@@ -0,0 +1,8 @@
+typos: misspelled identifiers correction:
```

Comment:
```suggestion
Typo in `{{ identifier }}`, fixes:
```

Reply:
I would mention `Possible` because it's not 100% accurate

Reply:
Ok